src/dashboard/debian/changelog

```diff
@@ -1,3 +1,10 @@
+archivematica-dashboard (1:1.3.2+4SNAPSHOT20150512174342-828eb-qa140~14.04) trusty; urgency=high
+
+  * commit: 828eb90be1a4bfe860b67a4398e8018456695dce
+  * 1.4 qa rc3 build
+
+ -- Justin Simpson <jsimpson@artefactual.com>  Tue, 12 May 2015 17:43:42 -0800
```

Comment:
in the changelogs,I think it might have to be the person who signed the package.  It is my gpg key that signed the files being uploaded to launchpad, that is based on my email address.

For copyright and control files, it should be Artefactual.

We could create an Artefactual gpg key but I am not sure if that is a good idea or not.  So far each individual uses their own gpg key.